RSWEB-9041 RSWEB-9042 RSWEB-9043 RSWEB-9045 RSWEB-9046: Resource center

brantdarilek opened the pull request:
**This PR requires RSWEB-9084!**
This PR covers RSWEB-9041 RSWEB-9042 RSWEB-9043 RSWEB-9045 RSWEB-9046. 

The best way to review the design is by reviewing each of the pages here:
http://rsweb-9041-9042-9043-9045-9046-resource-center.styleguide.dev.website.rackspace.com/derek/incubation/

runrog commented on fonts/config.json:
```diff
@@ -3049,6 +3049,76 @@
+      ]
+    },
+    {
+      "uid": "3b9dbe62ce1ded33cb5d6660c681bb08",
```
@brantdarilek I think you can remove this broken icon

brantdarilek replied:
Done.

WidgetsBurritos commented:
![screen shot 2017-05-03 at 11 04 09 pm](https://cloud.githubusercontent.com/assets/5263371/25690246/e23fb074-3054-11e7-91ff-a96c084b3619.png)
![screen shot 2017-05-03 at 11 04 15 pm](https://cloud.githubusercontent.com/assets/5263371/25690247/e2535de0-3054-11e7-9e94-8610187eafbe.png)

Some of the break points have overlap issues.  It almost looks intentional on some of the resources, but on videos specifically it could be a bit problematic.

WidgetsBurritos commented on styleguide/_layouts/default.jade:
```diff
@@ -18,7 +18,7 @@ html(lang="en")
     link(href="css/main.css", rel="stylesheet")
     link(href="https://fonts.googleapis.com/css?family=Open+Sans:400,600", rel="stylesheet")
-    link(href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.1/css/font-awesome.min.css" rel="stylesheet")
+    link(href="https://fonts.googleapis.com/css?family=Khand" rel="stylesheet")
```
Is it just me, or does Khand looks strange now?

Before: 
![screen shot 2017-05-03 at 11 29 17 pm](https://cloud.githubusercontent.com/assets/5263371/25690595/5f37064c-3058-11e7-81c5-f91871f6c955.png)

After:
![screen shot 2017-05-03 at 11 29 13 pm](https://cloud.githubusercontent.com/assets/5263371/25690596/5f3a8fe2-3058-11e7-947e-c2a25f10a849.png)

Maybe google just renders it differently or something, but this is definitely a big difference here that I want to make sure we're aware of, as it may create discrepancies between www and zoolander.

xinbin replied:
The first is definitely bold or something. It's thicker.

brantdarilek commented:
@WidgetsBurritos 
I've updated to accommodate for the tablet breakpoint. (See below images)
As for khand, I can't speak for the differences. I do think its better to include it than not. Not including it seems like more of a risk for discrepancies. 
@amberRrucker @runrog 
Do we host our own version on a CDN? If so I would just pull in that version instead of from google.

<img width="814" alt="screen shot 2017-05-04 at 10 17 07 am" src="https://cloud.githubusercontent.com/assets/799053/25710862/27b46e66-30b3-11e7-92a0-00a75465da6e.png">
<img width="833" alt="screen shot 2017-05-04 at 10 17 28 am" src="https://cloud.githubusercontent.com/assets/799053/25710863/27cb5536-30b3-11e7-9537-449d7acd134a.png">

runrog commented:
@brantdarilek don't think we have our own CDN link but in zoolander, try importing the rest of the Khand weights like this
```
<link href="https://fonts.googleapis.com/css?family=Khand:300,400,500,600,700" rel="stylesheet">
```
Looks like you are only including the standard weight

amberRrucker commented:
@brantdarilek We used to have it included but it was pulled out to avoid double loading the font into www. @WidgetsBurritos I'm preeeeetty sure that was your request but I dont remember.

WidgetsBurritos commented:
Looking into it a bit more, the discrepancy is between my local version of Khand and the version provided by google fonts.  I've disabled my local version of khand for now.  

I'd say keep the khand reference in here, but maybe make sure you have the necessary weights per Roger's comment, although I'm not sure we need all of those weights.  I honestly can't remember.  

Given that this will only affect zoolander (as www includes the font differently), I'm okay with it being a little less performant, as long as it provides consistency with what's in www. 

Otherwise I'm generally fine with this but do have one question.  I understand hiding the resource images on mobile, but does that make sense for the video resource?   There's no way to actually get to the video otherwise. 

![image](https://cloud.githubusercontent.com/assets/5263371/25712857/691c128c-30b8-11e7-8d1f-a5bc07970c2f.png)

brantdarilek commented:
@WidgetsBurritos @runrog @amberRrucker 
I've updated the font to include all weights.
I've also updated the template so not hide the video on mobile.

WidgetsBurritos commented:
@brantdarilek 

It's almost there.  It appears we still have a content overlap on from 992px to 1279px:

![image](https://cloud.githubusercontent.com/assets/5263371/25718800/0a5cc704-30cd-11e7-827f-c6af3d6c7db9.png)

brantdarilek commented:
@WidgetsBurritos 
Should be good now.

writes commented:
+1

WidgetsBurritos commented:
👍  so we can work on the www without much conflict.
